keyboards/controllerworks/city42/info.json

```diff
@@ -0,0 +1,131 @@
+            { "flags": 4, "matrix": [0, 10], "x": 202,  "y": 4 },
+            { "flags": 4, "matrix": [0, 9], "x": 187,  "y": 4 },
+            { "flags": 4, "matrix": [0, 8], "x": 172,  "y": 6 },
+            { "flags": 4, "matrix": [0, 7], "x": 158,  "y": 11 },
+            { "flags": 4, "matrix": [0, 6], "x": 144,  "y": 17 }
+          ]
+    },
+    "community_layouts": ["split_3x6_3"],
+    "layouts": {
+        "LAYOUT_split_3x6_3": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0, "y": 0.4 },
+                { "matrix": [0, 1], "x": 1, "y": 0.4 },
+                { "matrix": [0, 2], "x": 2, "y": 0.125 },
+                { "matrix": [0, 3], "x": 3, "y": 0 },
+                { "matrix": [0, 4], "x": 4, "y": 0.125 },
+                { "matrix": [0, 5], "x": 5, "y": 0.25 },
+                { "matrix": [0, 6], "x": 8, "y": 0.25 },
+                { "matrix": [0, 7], "x": 9, "y": 0.125 },
+                { "matrix": [0, 8], "x": 10, "y": 0 },
+                { "matrix": [0, 9], "x": 11, "y": 0.125 },
+                { "matrix": [0, 10], "x": 12, "y": 0.4 },
+                { "matrix": [0, 11], "x": 13, "y": 0.4 },
+                { "matrix": [1, 0], "x": 0, "y": 1.4 },
+                { "matrix": [1, 1], "x": 1, "y": 1.4 },
+                { "matrix": [1, 2], "x": 2, "y": 1.125 },
+                { "matrix": [1, 3], "x": 3, "y": 1 },
+                { "matrix": [1, 4], "x": 4, "y": 1.125 },
+                { "matrix": [1, 5], "x": 5, "y": 1.25 },
+                { "matrix": [1, 6], "x": 8, "y": 1.25 },
+                { "matrix": [1, 7], "x": 9, "y": 1.125 },
+                { "matrix": [1, 8], "x": 10, "y": 1 },
+                { "matrix": [1, 9], "x": 11, "y": 1.125 },
+                { "matrix": [1, 10], "x": 12, "y": 1.4 },
+                { "matrix": [1, 11], "x": 13, "y": 1.4 },
+                { "matrix": [2, 0], "x": 0, "y": 2.4 },
+                { "matrix": [2, 1], "x": 1, "y": 2.4 },
+                { "matrix": [2, 2], "x": 2, "y": 2.125 },
+                { "matrix": [2, 3], "x": 3, "y": 2 },
+                { "matrix": [2, 4], "x": 4, "y": 2.125 },
+                { "matrix": [2, 5], "x": 5, "y": 2.25 },
+                { "matrix": [2, 6], "x": 8, "y": 2.25 },
+                { "matrix": [2, 7], "x": 9, "y": 2.125 },
+                { "matrix": [2, 8], "x": 10, "y": 2 },
+                { "matrix": [2, 9], "x": 11, "y": 2.125 },
+                { "matrix": [2, 10], "x": 12, "y": 2.4 },
+                { "matrix": [2, 11], "x": 13, "y": 2.4 },
+                { "matrix": [3, 3], "x": 3.5, "y": 3.25 },
+                { "matrix": [3, 4], "x": 4.5, "y": 3.5 },
+                { "matrix": [3, 5], "x": 5.5, "y": 3.75, "h":1.5 },
+                { "matrix": [3, 6], "x": 7.5, "y": 3.75, "h":1.5 },
+                { "matrix": [3, 7], "x": 8.5, "y": 3.5 },
+                { "matrix": [3, 8], "x": 9.5, "y": 3.25 }
+            ]
+          }
```

Comment:
```suggestion
        }
```

Reply:
I think this was resolved in the info.json rewrite.